backend/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "watercool-backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.ts",
+  "scripts": {
+    "test": "npm run test",
+    "start": "tsc && node server.js"
```

Comment:
let's add package `nodemon` (only for development) & a script called "dev" that will run it
it automatically reruns node server onchange of any files